### PR TITLE
Rmaharjan/add tensorflow commit

### PR DESCRIPTION
**Pull request by rikenm1**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
- Adding missing TensorFlow dependencies. 

###### Change Log  

- Change : Add Tensorboard, Tensorboard plugins, and add missing runtime dep for python3-virtualvenv spec

###### Does this affect the toolchain?  

**YES/NO**
NO

###### Test Methodology

- Pipeline build id: [AMD] [286180](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=286180&view=results)
[ARM]: [286058](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=286058&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=d723e8ff-9fe0-5a1d-bed2-08b8dc44a520)

### CODE REVIEW

File: SPECS/python-distlib/python-distlib.spec [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+users then benefit from standardised implementation of low-level functions,
+time saved by not having to reinvent wheels, and improved interoperability
+between tools.
+
+%prep
+%setup -q -n %{srcname}-%{version}
+
+rm distlib/*.exe
+
+%generate_buildrequires
```
Comment (oliviacrain) [inline SPECS/python-distlib/python-distlib.spec:46]: We don't support this tag with our build system- please remove.

File: SPECS/python-distlib/python-distlib.spec [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+%pyproject_save_files %{srcname}
+
+%if %{with check}
+%check
+export PYTHONHASHSEED=0
+# test_sequencer_basic test fails due to relying
+# on the ordering of the input, hence disabling it.
+# https://github.com/pypa/distlib/issues/161
+%pytest -k "not test_sequencer_basic"
+%endif # with_tests
```
Comment (oliviacrain) [inline SPECS/python-distlib/python-distlib.spec:63]: No need for the `%if` guard

```suggestion
%check
export PYTHONHASHSEED=0
# test_sequencer_basic test fails due to relying
# on the ordering of the input, hence disabling it.
# https://github.com/pypa/distlib/issues/161
%pytest -k "not test_sequencer_basic"
```

File: SPECS/python-platformdirs/python-platformdirs.spec [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+Summary:        %{summary}
+BuildRequires:  python3-devel
+
+%description -n python3-%{srcname} %{common_description}
+
+%prep
+%autosetup -n %{srcname}-%{version}
+
+
+%generate_buildrequires
```
Comment (oliviacrain) [inline SPECS/python-platformdirs/python-platformdirs.spec:33]: Same comment as above- please remove

**Review by oliviacrain — APPROVED** [review]
Some nits, but overall LGTM

File: SPECS/python-platformdirs/python-platformdirs.spec (hunk 3 above)
Comment (rikenm1) [reply SPECS/python-platformdirs/python-platformdirs.spec]: removed

**Pull request merged**